js/foundation.orbit.js

```diff
@@ -344,11 +344,10 @@ class Orbit extends Plugin {
 
       if (this.options.useMUI && !this.$element.is(':hidden')) {
         Motion.animateIn(
-          $newSlide.addClass('is-active').css({'position': 'absolute', 'top': 0}),
+          $newSlide.addClass('is-active'),
           this.options[`animInFrom${dirIn}`],
           function(){
-            $newSlide.css({'position': 'relative', 'display': 'block'})
-            .attr('aria-live', 'polite');
+            $newSlide.attr('aria-live', 'polite');
```

Comment:
Slides are hidden when `motion-ui` is not used.

https://github.com/DanielRuf/foundation-sites/blob/dbdae96312bcccd0820d8ee43c088d5d6a1fab14/js/foundation.orbit.js#L153

We should keep `'display': 'block'`;

Reply:
But this causes the jump then.

Reply:
Ah, just the block property?

Reply:
Yeah, and remove `'position': 'relative'` from line 153. But make some tests (with and without motionUI / `mui` classes), I'm not sure.